src/logic/referral/referral.ts

```diff
@@ -233,6 +292,61 @@ export async function createReferralComponent(
         return
       }
 
+      if (acceptedInvites === TIERS_IRL_SWAG) {
+        try {
+          await slack.sendMessage({
+            channel: isDev ? 'notifications-dev' : 'referral-notifications',
+            text: `🎉 Referral Milestone Reached! - 100 Invites Tier Achieved by referrer ${referrer}`,
+            blocks: [
+              {
+                type: 'header',
+                text: {
+                  type: 'plain_text',
+                  text: '🎉 Referral Milestone Reached!',
+                  emoji: true
+                }
+              },
+              {
+                type: 'section',
+                text: {
+                  type: 'mrkdwn',
+                  text: `*🎯 100 Invites Tier Achieved!*\n\n*Referrer Wallet:* \`${referrer}\``
+                }
+              },
+              {
+                type: 'section',
+                text: {
+                  type: 'mrkdwn',
+                  text: `*📊 Details:*\n• Referrer has successfully invited 100 users\n• Tier milestone unlocked\n• Ready for outreach and communication\n\n*💬 Next Steps:*\n• Monitor for email submission from referrer`
+                }
+              },
+              {
+                type: 'actions',
+                elements: [
+                  {
+                    type: 'button',
+                    text: {
+                      type: 'plain_text',
+                      text: 'View Referral Dashboard',
+                      emoji: true
+                    },
+                    url: REFERRAL_METABASE_DASHBOARD,
+                    style: 'primary'
+                  }
+                ]
+              }
+            ]
+          })
```

Comment:
In order no to have so many lines defined in the component, what do you think about moving the message to a utils file that builds it? The same goes for the rejection one.